### PR TITLE
Week 1 Module 1 Portfolio: mobile version

### PR DESCRIPTION
**Pull request by the-catalystmc**
In this project I:
Built the middle pages of the mobile website.
Added the necessary buttons.
Add hover effect etc to buttons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
+            </div>
+
+            <div class="lb">
+                <button class="large-button" type="button">See Project</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Good job using the `<section>` tag for Semanting HTML concepts. However, this `<section>` should be the wrapper of the work's project cards. In the project requirements, the work's section is wrapping a `<ul>` tag for all the projects, and each project should be wrapped inside the `<li>` tag as an element of this list (`<ul>`). Kindly let's work on that 👍

File: index.html [hunk 2]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+                </ul>
+            </div>
+
+            <p class="primary-read">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
+            </div>
```
Comment (MahmoudBakr23) [inline index.html]: - [x] In order to continue following the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) concepts. Please keep in mind that this part of the project's card should be a list (`<ul>` tag) on its own, and each element is a `<li>` tag as a list-item, and not a button! Kindly fix this issue here.

File: index.html [hunk 3]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <img class="projectpic" src="images/project1.svg" alt="project snapshot">
+            <h2>Tonic</h2>
+            <div class="canopy-flex">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li class="back-end">•  Back End Dev  •</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Please notice that there should be no blank lines or indentations unless there's a good reason for them to be. It's way too better without them in keeping the code clean and organized.

> Kindly remember that the blank lines and indentations are only used to separate two logical pieces of code or two large sections apart.

File: index.html [hunk 4]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <p class="primary-read">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 5]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <h2 class>Multi-Post
+                Stories</h2>
+            <div class="canopy-flex">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li class="back-end">•  Back End Dev  •</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 6]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <p class="primary-read">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 7]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <div class="lb">
+                <button class="large-button" type="button">See Project</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 8]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <img class="projectpic" src="images/project3.png" alt="project snapshot">
+            <h2>Tonic</h2>
+            <div class="canopy-flex">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li class="back-end">•  Back End Dev  •</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 9]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <p class="primary-read">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 10]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <div class="lb">
+                <button class="large-button" type="button">See Project</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 11]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <h2 class>Multi-Post
+                Stories</h2>
+            <div class="canopy-flex">
+                <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li class="back-end">•  Back End Dev  •</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 12]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <p class="primary-read">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 13]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+            <div class="lb">
+                <button class="large-button" type="button">See Project</button>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 14]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+                <img src="images/css.svg" alt="javascript">
+                <h5 class="title-text">CSS</h5>
+            </section> 
+            <div class="qualification">
+                <ul class="seperator">
+                    <li><h3>Frameworks</h3></li>
+                <li><img src="images/Union.svg" alt="arrow"></li>
+                </ul>
+            </div>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 15]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+                <ul class="seperator">
+                <li><h3>Skills</h3></li>
+                <li><img src="images/Union.svg" alt="arrow"></li>
+            </ul>
+            </div>
+            
+                </section>
+                
+
+
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Let's remove this blank line and the extra space (indentations) here as well.

_Please look at the comment about the blank lines above ⬆️_

File: index.html [hunk 16]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+
+            <p class="primary-read">A daily selection of privately personalized reads;
+                no accounts or sign-ups required.</p>
+            <div>
+                <button class="small-button" type="button">html</button>
+                <button class="small-button" type="button">css</button>
+                <button class="small-button" type="button">javaScript</button>
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Please adjust this part here to a `<ul>` tag with `<li>` tag for each element.

_Kindly look at the comment about the Semantic HTML on that part up ⬆️_

File: index.html [hunk 17]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Please adjust this part here to a `<ul>` tag with `<li>` tag for each element.

_Kindly look at the comment about the Semantic HTML on that part up ⬆️_

File: index.html [hunk 18]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
```
Comment (MahmoudBakr23) [inline index.html]: - [x] Please adjust this part here to a `<ul>` tag with `<li>` tag for each element.

_Kindly look at the comment about the Semantic HTML on that part up ⬆️_

File: index.html [hunk 19]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
...
+                <img src="images/html.svg" alt="javascript">
+                <h5 class="title-text">HTML</h5>
+            </section>
+            <section class="lang-container">
+                <img src="images/css.svg" alt="javascript">
+                <h5 class="title-text">CSS</h5>
+            </section> 
+            <div class="qualification">
+                <ul class="seperator">
+                    <li><h3>Frameworks</h3></li>
```
Comment (MahmoudBakr23) [inline index.html]: - I can see the great effort you put into this block of code here. However, there are a few adjustments that need to be implemented to make the code more organized, readable from the browsers, and optimized. Kindly consider the following.

- [ ] Please make sure to wrap the categories ("Languages", "Frameworks", "Skills") inside a `<ul>` tag with each one of them a `<li>` tag.
- [x] Then, the first `<li>` tag which is the "Languages" will have inside of it a second smaller `<ul>` tag which ("Javascript", "HTML", "CSS") will be its `<li>` tags, one for each.

Kindly let's work on that here 👍

File: index.html [hunk 20]
```diff
@@ -42,8 +42,174 @@ <h6 class="connect"><a href="#">LET’S CONNECT</a></h6>
                 <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorpeace.svg"
                             alt="social media image"></a></li>
             </ul>
-
         </div>
+        <!--PROJECTS SECTION BEGIN HERE -->
```
Comment (MahmoudBakr23) [inline index.html:46]: **Highlight**
This is such a great thing you did by writing all of the clarification comments in the HTML and CSS files. It helped a lot 👍 Keep up 💪

File: style.css [hunk 21]
```diff
@@ -101,3 +109,215 @@ body {
...
+  padding: 16px;
+  width: 375px;
+  height: 650px;
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin-bottom: 100px;
+}
```
Comment (MahmoudBakr23) [inline style.css]: - [x] Please make sure to make the wrapper section of the projects is styled by the CSS grid, and each project's card within that section is styled by the CSS flexbox.

> Kindly notice that it's mandatory that you use both grid and flexbox of styling this section. You can always refer to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

- [x] Also, following the design template on the background colors on this section. Please refer to the design template and pay attention to the background colors between the whole section and each project's card.

File: style.css [hunk 22]
```diff
@@ -101,3 +109,215 @@ body {
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  margin-bottom: 50px;
+}
```
Comment (MahmoudBakr23) [inline style.css]: - [x] Please follow the exact same styling as the given design such as font typography, font size, font weight, margin, and padding.
Let's take a look at the following two images and see the differences.
### **Your design**
Screenshot 3 student
### **Original design**
Screenshot 3

File: style.css [hunk 23]
```diff
@@ -101,3 +109,215 @@ body {
...
+  display: flex;
+  color: #000;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lang-container {
+  display: flex;
+  width: 303px;
+  height: 66.72px;
```
Comment (MahmoudBakr23) [inline style.css]: - [x] Please apply the corresponding changes for this section to match the given design.

> Kindly look up the comment about the given design modifications for more details.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @the-catalystmc 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is well-configured 💯 
- Descriptive PR headline, summary, and commit messages 💯 

_Please check the **Highlight** comments under the review._

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] In the future, please remember to commit every single completed change to the GitHub repo.
Kindly notice that the benefit of that is professional by making you organized, and technical by making you avoid losing important pieces of code that haven't been committed yet.

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the **OPTIONAL** comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (the-catalystmc) [reply index.html]: Hello, how do you see this on Figma? I select the content but there is no such detail.

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @the-catalystmc, 

## You have done a great job of addressing all the issues that the previous reviewer pointed out. It reflects excellent efforts. Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**